packages/scaffold/src/partials/w3m-email-login-widget/styles.ts

```diff
@@ -24,7 +24,7 @@ export default css`
   wui-icon-link,
   wui-loading-spinner {
     position: absolute;
-    top: 50%;
+    top: 26px;
```

Comment:
If I remember correctly hardcoding this value caused layout issues in safari or firefox, could you validate that everything looks as expected there? If possible we should use dynamic values like these to account for potential ui component size changes in the future.

Reply:
hm fair enough, will double check. 

This was changed because when the input has an error text below, it's height grows, and the 50% makes it so the icon slides out of the input. 

Maybe there's a way to move the relative point on the input so it properly considers the icon positioning when errors occur. 
Maybe there's a container where we can fix its height + make it relative and would fix it elegantly